src/component/absensi/employee/TableEmp.vue

```diff
@@ -1,67 +1,27 @@
 <template>
-    <div>
-        <table>
-            <thead>
-            <tr>
-
-                <td>NIK
-                <td>Full Name</td>
-                <td>Gender</td>
-                <td>Position Text</td>
-                <td>Organizational Unit</td>
-                <td>Marital Status</td>
-                <td>Religion</td>
-                <td>Name Of Dept</td>
-                <td>Chief NIK</td>
-                <td>Chief Name</td>
-                <td>Start Working Date</td>
-            </tr>
-            </thead>
-            <tbody>
-            <tr>
-            <tr>
-
-                <td>NIK
-                <td>Full Name</td>
-                <td>Gender</td>
-                <td>Position Text</td>
-                <td>Organizational Unit</td>
-                <td>Marital Status</td>
-                <td>Religion</td>
-                <td>Name Of Dept</td>
-                <td>Chief NIK</td>
-                <td>Chief Name</td>
-                <td>Start Working Date</td>
-            </tr>
-
-            </tbody>
-        </table>
-    </div>
-
+    <tr>
+        <td>{{item.nik}}</td>
+        <td>{{item.fullname}}</td>
+        <td>{{item.gender}}</td>
+        <td>{{item.positionText}}</td>
+        <td>{{item.organizationalUnitText}}</td>
+        <td>{{item.maritalStatus}}</td>
+        <td>{{item.religion}}</td>
+        <td>{{item.nameOfDept}}</td>
+        <td>{{item.chiefNik}}</td>
+        <td>{{item.chiefName}}</td>
+        <td>{{item.startWorkingDate}}</td>
+    </tr>
 </template>
```

Comment:
Let's discuss it. First, I want to know why did you separate the table in the first place. Why did you separate them?